charts/local-kubevirt/update.bash

```diff
@@ -56,11 +56,15 @@ wget https://github.com/kubevirt/containerized-data-importer/releases/download/$
 wget https://github.com/kubevirt/containerized-data-importer/releases/download/$latest_cdi_release/cdi-cr.yaml -O ${dir}/templates/cdi-cr.yaml
 
 pushd $dir/templates
+sed -i -e "s/\(^[ ]*namespace:\) cdi$/\1 kubevirt/" cdi-operator.yaml
+
 yq -s '.kind + "-" + .metadata.name + ".yaml"' ./kubevirt-operator.yaml
 rm ./kubevirt-operator.yaml
+rm ./Namespace-kubevirt.yaml
 
 yq -s '.kind + "-" + .metadata.name + ".yaml"' ./cdi-operator.yaml
 rm ./cdi-operator.yaml
+rm ./Namespace-cdi.yaml
```

Comment:
for the local installation, we just used the upstream manifests and "helmified" them. But no `Namespaces` should be part of a helm chart.